src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -107,11 +145,16 @@ export class ContentTypeFieldsDropZoneComponent implements OnInit, OnChanges, On
         this.fieldDragDropService.fieldDropFromSource$
             .pipe(takeUntil(this.destroy$))
             .subscribe((data: DropFieldData) => {
-                this.setDroppedField(data.item);
-                this.toggleDialog();
+                if (data.item.clazz === COLUMN_BREAK_FIELD.clazz) {
+                    setTimeout(() => {
+                        this.saveFields.emit(splitRows(this.fieldRows));
+                    }, 0);
```

Comment:
maybe we should create a method for this, I think this exactly code is repeat in another place